Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:15.12.0-alpine
+ENV NODE_ENV dev
+
+# Add a work directory
+WORKDIR /app
+
+# Cache and Install dependencies
+COPY package.json .
+RUN npm install
+
+# Copy app files
+COPY . .
+
+# Expose port
+EXPOSE 3000
+
+# Start the app
+CMD ["npm", "start"]
```

Comment:
Let's use nginx as the webserver. You can read how it is beneficial to do so [here](https://medium.com/intrinsic-blog/why-should-i-use-a-reverse-proxy-if-node-js-is-production-ready-5a079408b2ca).

To do this, you'll need to set up a [multi-staged docker build](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds)

Reply:
Done.